app/styles/app.scss

```diff
@@ -43,6 +43,11 @@ body {
   .article {
     @include newspaper;
 
+    .highlighted {
+      // TODO: allow topic switching and don't just highlight one color
+      background-color: $red_accent;
+    }
+
     /* react makes this not work */
     &[data-topic="1"]::selection {
```

Comment:
this CSS code is sort of unused, but the answer to that is yes and no - yes because you can't select any data topics right now (which will be changed soon), but no because if you could, you'd be able to select any of the `data-topics`

also this use of the `data-*` attribute will be replaced by react